roles/create_rds_global_cluster/tasks/create.yml

```
@@ -0,0 +1,50 @@
+---
+- name: Run 'create_rds_global_cluster' role create operations
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+
+  block:
+    - name: Create rds global database
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"
+        engine: "{{ create_rds_global_cluster_enigne  | default('aurora-postgresql')}}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        state: present
+
+    - name: Create a primary cluster for global database "{{ create_rds_global_cluster_global_cluster_name }}" in "{{ create_rds_global_cluster_primary_cluster_region }}"
+      amazon.aws.rds_cluster:
+        db_cluster_identifier: "{{ create_rds_global_cluster_primary_cluster_name | default('{{ create_rds_global_cluster_global_cluster_name }}') }}" #aka cluster_name
+        region: "{{ create_rds_global_cluster_primary_cluster_region }}"
+        engine: "{{ create_rds_global_cluster_enigne }}"
+        engine_version: "{{ create_rds_global_cluster_engine_version | default(omit) }}"
+        username: "{{ create_rds_global_cluster_username }}"
+        password: "{{ create_rds_global_cluster_password }}"
+        db_subnet_group_name: "{{ create_rds_global_cluster_db_subnet_group_name | default(omit) }}"
+        global_cluster_identifier: "{{ create_rds_global_cluster_global_cluster_name }}"  # what global db cluster should be associated to
+      ignore_errors: true
```

Why are you ignoring errors here?

---

left over from debugging. removed.